src/message/index.js

```diff
@@ -0,0 +1,4 @@
+export * from './default.component';
```

Comment:
it's a very bad pattern to named files with components with dot character.
Please rename all corresponding files just like component name inside this file(of course file name must be with capital letter).

_src/message/FunctionalComponent.js
src/message/PureComponent.js_
_....._

Reply:
Oh. Actually I followed the naming convention from angular. Will make the changes as per your feedback.